### ISSUE TITLE
v100 推理卡，图片文字较多时，推理时间10秒左右一张（着急！！！）

### ISSUE THREAD

**Issue by githublsk** [opendatalab/MinerU#4205]
### 🔎 Search before asking | 提交之前请先搜索

- [x] I have searched the MinerU [Readme](https://github.com/opendatalab/MinerU) and found no similar bug report.
- [x] I have searched the MinerU [Issues](https://github.com/opendatalab/MinerU/issues) and found no similar bug report.
- [x] I have searched the MinerU [Discussions](https://github.com/opendatalab/MinerU/discussions) and found no similar bug report.

### 🤖 Consult the online AI assistant for assistance | 在线 AI 助手咨询

- [x] I have consulted the [online AI assistant](https://deepwiki.com/opendatalab/MinerU) but was unable to obtain a solution to the issue.

### Description of the bug | 错误描述

torch:2.9.0
vllm:0.11.2
CUDA:12.4
显卡：v100 32G
启动服务的命令：vllm serve opendatalab/MinerU2.5-2509-1.2B --host 127.0.0.1 --port 8000 \
  --logits-processors mineru_vl_utils:MinerULogitsProcessor --gpu_memory_utilization 0.8
客户端测试代码：
from PIL import Image
from mineru_vl_utils import MinerUClient

client = MinerUClient(
    backend="http-client",
    server_url="http://127.0.0.1:8000"
)

image = Image.open("/path/to/the/test/image.png")
extracted_blocks = client.two_step_extract(image)
print(extracted_blocks)

问题描述：单张图片推理时间从2-20秒之间不等，推理时间和图片内文字多少有关系，感觉推理时间太慢了，请问是什么问题？谢谢回复！

### How to reproduce the bug | 如何复现

torch:2.9.0
vllm:0.11.2
CUDA:12.4
显卡：v100 32G
启动服务的命令：vllm serve opendatalab/MinerU2.5-2509-1.2B --host 127.0.0.1 --port 8000 \
  --logits-processors mineru_vl_utils:MinerULogitsProcessor --gpu_memory_utilization 0.8
客户端测试代码：
from PIL import Image
from mineru_vl_utils import MinerUClient

client = MinerUClient(
    backend="http-client",
    server_url="http://127.0.0.1:8000"
)

image = Image.open("/path/to/the/test/image.png")
extracted_blocks = client.two_step_extract(image)
print(extracted_blocks)

问题描述：单张图片推理时间从2-20秒之间不等，推理时间和图片内文字多少有关系，感觉推理时间太慢了，请问是什么问题？谢谢回复！

### Operating System Version| 操作系统版本

torch:2.9.0
vllm:0.11.2
CUDA:12.4
显卡：v100 32G
启动服务的命令：vllm serve opendatalab/MinerU2.5-2509-1.2B --host 127.0.0.1 --port 8000 \
  --logits-processors mineru_vl_utils:MinerULogitsProcessor --gpu_memory_utilization 0.8
客户端测试代码：
from PIL import Image
from mineru_vl_utils import MinerUClient

client = MinerUClient(
    backend="http-client",
    server_url="http://127.0.0.1:8000"
)

image = Image.open("/path/to/the/test/image.png")
extracted_blocks = client.two_step_extract(image)
print(extracted_blocks)

问题描述：单张图片推理时间从2-20秒之间不等，推理时间和图片内文字多少有关系，感觉推理时间太慢了，请问是什么问题？谢谢回复！

**Comment by githublsk** [comment]
@myhloli 麻烦帮忙回复一下啊，十分感谢

**Comment by myhloli** [comment]
速度正常的，生成式模型速度和token数量成正比的

**Comment by githublsk** [comment]
> 速度正常的，生成式模型速度和token数量成正比的

你的意思是V100我测试的这个速度是正常现象？？

**Comment by myhloli** [comment]
v100本身也挺老的了，如果你换a100或者h100那是能快不少的

**Comment by Liang200018** [comment]
> 速度正常的，生成式模型速度和token数量成正比的

一些图片会识别方向颠倒，进而导致生成很多重复的，请问，这种可以限制生成长度吗，这种重复的生成是错的，直接中断可以吗
